public-keys-component.html

```diff
@@ -0,0 +1,14 @@
+<dt>Cryptographic Keys</dt>
+<dd>
+  <ul class="list-unstyled">
+    <li ng-show="$ctrl.identity.publicKey.length > 0" ng-repeat="key in $ctrl.identity.publicKey">
```

Comment:
Is this `ng-show` necessary? If the length is 0, doesn't it not show an `<li>` automatically?